mocks/data/submit/baseMissingAttachments.ts

```diff
@@ -0,0 +1,160 @@
+const capitatedAmendmentBase = {
```

Comment:
This file causes the zod errors that get caught in each particular submission event

Reply:
could you re-use the base amendments and use override the attachments for the tests? like 
```
{
    ...base.capitatedAmendmentBase,
    attachments: {}
}
```